examples/svg.rs

```diff
@@ -0,0 +1,19 @@
+fn main() {
+    #[cfg(feature = "svg")]
```

Comment:
Do we need to specify the #[cfg] attribue here ?

Reply:
Well the examples don't build as the features aren't enabled by default, technically it doesn't matter, I just get an error in my editor which is why I added it

Reply:
Hmm, I don't know how to prevent people from running the example without the specified features ?

Maybe you have less error by doing so:

```rust
#[cfg(feature = "svg")]
fn main() {
  ...
}
```

And not providing a default main? It might be wierd to the user tho

Reply:
The examples need a main so that doesn't work either 😅 

We could maybe add a print to explain that they need to enable the feature if its not enabled?

Reply:
Doesn't remove the errors, sadly
```
`main` function not found in crate `svg`
consider adding a `main` function to `examples\svg.rs`
```

Reply:
Yeah, a print or a panic, which would be better in my opinion? It's up to you :)

Reply:
Maybe adding 2 mains one for the feature and one for not using it?

Reply:
Having two mains, is more readable, I agree